src/frontend/scripts/cookie-panel.es6

```diff
@@ -223,10 +207,12 @@
       showCookiePanelSettings();
     });
 
-    document.getElementById("cookie-panel-banner-reject-button").addEventListener("click", () => {
-      document.getElementById("cookie-panel-banner").style.display = "none";
-      rejectAllCookies();
-    });
+    if (config.rejectLabel) {
```

Comment:
Yeah, I thought about that as well but decided not to because this way we can upgrade any of our projects that already are using the cookie panel without any issue. 

Never know if the customer doesn't actually want any "Reject all" button (I'm not sure on the norwegian GDPR implementation if such a button is mandatory or not to be honest, it wasn't in Belgium at least 😅 ). A lot of projects use custom styling which might be missing proper styling for that new button, like FFI. In those cases it would be shown unstyled until we update the project itself, which is just another small annoyance.